Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -450,7 +615,8 @@ public class HTTPClient {
                     maximumAllowedIdleTimeInConnectionPool: TimeAmount,
                     proxy: Proxy? = nil,
                     ignoreUncleanSSLShutdown: Bool = false,
-                    decompression: Decompression = .disabled) {
+                    decompression: Decompression = .disabled,
+                    backgroundActivityLogger: Logger?) {
```

Comment:
I'm losing the thread, is it not used anywhere?

Reply:
@avolokhov whoops, sorry, leftover from a transitional state. Fixed now.